packages/editor-ui/src/views/NodeView.vue

```diff
@@ -683,13 +683,13 @@ export default mixins(
 				}
 
 				if (e.key === 'd') {
-					this.callDebounced('deactivateSelectedNode', 350);
+					this.callDebounced('deactivateSelectedNode', { debounceTime: 350, trailing: false });
```

Comment:
we don't need to specify trailing is false, right? we can simply remove it since default is leading

Reply:
Right. Updated.